mbs-s0003-20230501/README.md

```diff
@@ -0,0 +1,140 @@
+# Map-based noise simulations based on `scan-s0003` time-domain simulations
+
+Tag: `mbs-s0003-20230501`
+
+## Updates
+
+* 2023-05-09: first release, `lmax=5400` simulations.
+
+## Summary
+
+300 realistic realizations of the map noise, given the single realization of the time-domain simulations. Simulations cover the full LAT footprint to a bandlimit of `lmax=5400`, and utilize the directional wavelet noise model.
+
+## Noise model
+
+This release is based on the `scan-s0003` time-domain simulations, available on `NERSC` at:
+
+    /global/cfs/cdirs/sobs/sims/scan-s0003/output/combined/processed/fejer1
+
+These time-domain simulations implement the type-1.2.1 high-cadence 3-elevation scan-strategy (see scan-strategy [repo](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/scan-strategy-sims-2023) and [ICD](https://docs.google.com/presentation/d/1hInPLZYGQrO1E7j9FxR6UlH5RZarJsQ3mDHzlU0i7I8/edit#slide=id.p)). These maps provide a single realization of the noise in each split of each frequency band. 
+
+Using [`mnms`](https://github.com/simonsobs/mnms) (commit `d4afb9d`), we generate 300 realizations of the noise using map-based methods. Specifically, we use the directional wavelet model, governed by the configured [parameters](parameters/so_scan_s0003.yaml).
```

Comment:
is it possible to tag a release of `mnms` at that commit?

Reply:
It will be, but not immediately. These sims required some new developments, and I ran them as soon as it was working for this model, but the code is not properly implemented yet for the other models, and is a bit messy. (For this reason, for instance, I am only directing users to load the sims using the low-level `pixell.enmap.read_map` function, rather than from within `mnms`, which would otherwise be possible and would handle filename management automatically). Since there won't be any need to install or use `mnms` to load these sims at this point, I included this commit reference only for record-keeping. I will be able to release a new version soon though, and I can update these mbs release details then if that works?

Reply:
yes, sure